src/main/java/Task.java

```diff
@@ -0,0 +1,51 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public static Task taskCon(String userInput) throws InvalidCommandException, InvalidTaskCreationException{
```

Comment:
Would you like to rename this variable's 'con' term to an actual word, and include JavaDoc? I'm sure it will be helpful to readers!